### ISSUE TITLE
Referral from ARIA2 to 3.3.3 should be 3.3.1

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1802]
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA2

> Applicability
> Technologies that support Accessible Rich Internet Applications (WAI-ARIA).
> 
> This technique relates to:
> 
> Success Criterion 1.3.1: Info and Relationships (Advisory)
> Success Criterion 3.3.3: Error Suggestion (Sufficient)

"Success Criterion 3.3.3: Error Suggestion" (Sufficient) should be "Success Criterion 3.3.1: Error Identification"
